### PR TITLE
Update Algorand Wallets list and explorer

### PR DESCRIPTION
**Pull request by envin3**
- Update algorand explorer
- Remove deprecated wallets

### CODE REVIEW

File: src/utils/__tests__/explorer.test.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ describe('getCorrespondingTokenExplorerLinkByBlockchain', () => {
     ['ULTRA', 'https://explorer.mainnet.ultra.io/account/tokenaddress'],
     ['ARBITRUM', 'https://arbiscan.io/token/tokenaddress'],
     ['LUXOCHAIN', 'https://explorer.luxochain.io/token/tokenaddress'],
-    ['ALGORAND', 'https://algoexplorer.io/asset/tokenaddress'],
+    ['ALGORAND', 'https://allo.info/asset/tokenaddress'],
```
Comment (oliviera9) [inline src/utils/__tests__/explorer.test.js:48]: I think this is wrong.
This is a working link for instance https://allo.info/asset/748211185/token/

File: src/utils/__tests__/explorer.test.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ describe('getCorrespondingTxExplorerLinkByBlockchain', () => {
     ['ULTRA', 'https://explorer.mainnet.ultra.io/tx/txhash'],
     ['ARBITRUM', 'https://arbiscan.io/tx/txhash'],
     ['LUXOCHAIN', 'https://explorer.luxochain.io/tx/txhash'],
-    ['ALGORAND', 'https://algoexplorer.io/tx/group/txhash'],
+    ['ALGORAND', 'https://allo.info/tx/group/txhash'],
```
Comment (oliviera9) [inline src/utils/__tests__/explorer.test.js:22]: Check also this please. See following comment.

**Review by oliviera9 — COMMENTED** [review]
I think this PR needs further checks.

File: src/utils/__tests__/explorer.test.js (hunk 1 above)
Comment (oliviera9) [reply src/utils/__tests__/explorer.test.js:48]: Indeed, https://allo.info/asset/tokenaddress/748211185 does not work

Comment (oliviera9) [reply src/utils/__tests__/explorer.test.js:48]: Stick to test-driven development: fix unit tests expected values, then adapt the impl, not viceversa.

Comment (envin3) [reply src/utils/__tests__/explorer.test.js:48]: The expected link is of the type https://allo.info/asset/748211185. In the test it is checked that `getCorrespondingTokenExplorerLinkByBlockchain` called with `blockchain` and `'tokenaddress'` returns  'https://allo.info/asset/tokenaddress' where `'tokenaddress'` is a mock for the actual address, not a part of the link.

If you check https://github.com/pnetwork-association/ptokens-dapp/blob/refactor/algorand/src/utils/explorer.js#L41 you can see indeed that with `748211185` the generated link is https://allo.info/asset/748211185.

File: src/utils/__tests__/explorer.test.js (hunk 2 above)
Comment (envin3) [reply src/utils/__tests__/explorer.test.js:22]: Here the target is `'https://allo.info/tx/txhash'`
Group should have been removed. Good catch

File: src/utils/__tests__/explorer.test.js (hunk 1 above)
Comment (oliviera9) [reply src/utils/__tests__/explorer.test.js:48]: > The expected link is of the type https://allo.info/asset/748211185. In the test it is checked that getCorrespondingTokenExplorerLinkByBlockchain called with blockchain and 'tokenaddress' returns 'https://allo.info/asset/tokenaddress' where 'tokenaddress' is a mock for the actual address, not a part of the link.

OK!

File: src/utils/explorer.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ const transactionBaseLink = {
   ULTRA: `${settings.explorers.mainnet.ultra}tx/%s`,
   ARBITRUM: `${settings.explorers.mainnet.arbitrum}tx/%s`,
   LUXOCHAIN: `${settings.explorers.mainnet.luxochain}tx/%s`,
-  ALGORAND: `${settings.explorers.mainnet.algorand}tx/group/%s`,
+  ALGORAND: `${settings.explorers.mainnet.algorand}tx/%s`,
```
Comment (oliviera9) [inline src/utils/explorer.js:21]: I am not that convinced about this.
Check this for instance:
https://allo.info/tx/GFR37YZMVMK7NGGLS32OU2JXIQEAAYCPYYGZOZ374MSM332RJTFQ/group
IIRC the result returned by the API contains the group ID.
This is an example: TDUi2peczKlp5TMoKUvxjvkKKwtdMPgcKbv9aJoSGx8=
and the link to this is
https://allo.info/tx/group/TDUi2peczKlp5TMoKUvxjvkKKwtdMPgcKbv9aJoSGx8%3D

Comment (oliviera9) [reply src/utils/explorer.js:21]: So probably the previous tx/group path is correct!

**Review by oliviera9 — APPROVED** [review]
LGTM

**Pull request merged**